Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/SANSILLReduction.py

```diff
@@ -61,6 +61,22 @@ def _check_distances_match(ws1, ws2):
         if fabs(l2_1 - l2_2) > tolerance:
             logger.warning('Different distances detected! {0}: {1}, {2}: {3}'.format(r1, l2_1, r2, l2_2))
 
+    @staticmethod
+    def _check_wavelengths_match(ws1, ws2):
+        """
+            Checks if the wavelength difference between the data is close enough
+            @param ws1 : workspace 1
+            @param ws2 : workspace 2
+        """
+        tolerance = 0.01 # A
+        wavelength_1 = ws1.getRun().getLogData('wavelength').value
```

Comment:
This works except in TOF, where the wavelength in the logs is 0. For TOF, transmissions and fluxes are functions of wavelength, so this check doesn't make sense. So the checks should be performed if it is not in TOF, there must be a flag somewhere in the algorithm already.

Reply:
Noticed and corrected, also added masking of the instrument edges, as requested, and fixed the release notes conflict.